docs/sdk/examples/pastefromgoogledocs.html

```diff
@@ -0,0 +1,199 @@
+
+					&lt;p style="text-align:center"&gt;&lt;span style="font-size:11pt"&gt;&lt;span style="font-family:Georgia"&gt;&lt;span style="color:#333333"&gt;&lt;em&gt;This new editor has totally changed our content creation experience!&lt;/em&gt;&lt;/span&gt;&lt;/span&gt;&lt;/span&gt;&lt;/p&gt;
+
+					&lt;p style="text-align:center"&gt;&lt;span style="font-size:11pt"&gt;&lt;span style="font-family:Georgia"&gt;&lt;span style="color:#333333"&gt;&lt;em&gt;&mdash; John F. Smith, CEO, The New Web&lt;/em&gt;&lt;/span&gt;&lt;/span&gt;&lt;/span&gt;&lt;/p&gt;
+
+					&lt;p style="text-align:justify"&gt;&lt;span style="font-size:11pt"&gt;&lt;span style="font-family:Georgia"&gt;&lt;span style="color:#333333"&gt;This letter recognizes that much of our success is directly attributable to your efforts. You deserve to be proud of your achievement. May your future efforts be equally successful and rewarding.&lt;/span&gt;&lt;/span&gt;&lt;/span&gt;&lt;/p&gt;
+
+					&lt;p style="text-align:justify"&gt;&lt;span style="font-size:11pt"&gt;&lt;span style="font-family:Georgia"&gt;&lt;span style="color:#333333"&gt;I am sure we will be seeing and hearing a great deal more about your accomplishments in the future. Keep up the good work!&lt;/span&gt;&lt;/span&gt;&lt;/span&gt;&lt;/p&gt;
+
+					&lt;p&gt;&nbsp;&lt;/p&gt;
+
+					&lt;p&gt;&lt;span style="font-size:11pt"&gt;&lt;span style="font-family:Georgia"&gt;&lt;span style="color:#333333"&gt;Best regards,&lt;/span&gt;&lt;/span&gt;&lt;/span&gt;&lt;/p&gt;
+
+					&lt;p&gt;&lt;span style="font-size:11pt"&gt;&lt;span style="font-family:Georgia"&gt;&lt;span style="color:#333333"&gt;&lt;em&gt;The Management&lt;/em&gt;&lt;/span&gt;&lt;/span&gt;&lt;/span&gt;&lt;/p&gt;
+				</textarea>
+			</div>
+		</div>
+
+		<h2>Related Features</h2>
+		<ul>
+			<li><a href="./pastefromword.html">Inserting Content &ndash; Pasting Content from Microsoft Word</a></li>
+			<li><a href="./pastefromexcel.html">Inserting Content &ndash; Pasting Content from Microsoft Excel</a></li>
+			<li><a href="./styles.html#widget-styles">Styling and Formatting &ndash; Applying Styles to Editor Content</a></li>
+			<li><a href="./colorbutton.html">Styling and Formatting &ndash; Setting Text and Background Color</a></li>
+			<li><a href="./acf.html">Content Filtering &ndash; Advanced Content Filter &ndash; Automatic Mode</a></li>
+		</ul>
+
+		<h2>Sample Google Docs Document</h2>
+
+		<ul>
+			<li><a href="https://docs.google.com/document/d/1eLN7jVMlR771M1goN4Hv1PCmZeqzs3daU09Nz1sjZiM/">"CKEditor 4 Google Docs Test" document</a></li>
+		</ul>
+
+		<script data-sample="1">
+			CKEDITOR.replace( 'editor1', {
+				// Define the toolbar: https://ckeditor.com/docs/ckeditor4/latest/features/toolbar
+				// The full preset from CDN which we used as a base provides more features than we need.
+				// Also by default it comes with a 3-line toolbar. Here we put all buttons in a single row.
+				toolbar: [
```

Comment:
Theoretically, we could add custom CSS for tables here (`width: 100%;`) so has same width after pasting as in the sample document:

![image](https://user-images.githubusercontent.com/1061942/65488762-1a62cd00-deaa-11e9-8c4c-fd94f7d26a69.png)

instead of:

![image](https://user-images.githubusercontent.com/1061942/65488799-28185280-deaa-11e9-975a-f329f62a3c6c.png)

but then it will break other custom tables pasting so I'm not sure here. WDYT? @Comandeer @AnnaTomanek

Reply:
I'd leave it as is. These custom styles would work only for this table in the sample and can make users think that something does not work correctly.